api/messages.go

```diff
@@ -69,6 +83,10 @@ func readJSONRequest(w http.ResponseWriter, r *http.Request, req interface{}) er
 	if ok && len(operatorType) > 0 && strings.EqualFold(operatorType[0], "solo") {
 		req.(*CreateCredentialRequest).operatorType = credentials.OperatorType(pb.OperatorType_OT_SOLO)
 	}
+	if !ok {
+		const msg = "missing required URL parameter: operator_type"
+		return &decodingError{status: http.StatusBadRequest, msg: msg}
+	}
```

Comment:
I think we should leave this as optional/defaulting to 0, so we don't break downstream integrations.

Academically, this is important... however, for us, we only expect 1 integration, but better to observe good practice.

Reply:
Works for me. I'll go ahead and remove the check.